test/Interop/Cxx/class/access/Inputs/non-public.h

```diff
@@ -0,0 +1,60 @@
+#ifndef NON_PUBLIC_H
+#define NON_PUBLIC_H
+
+// Override this to test structs
+#ifndef TEST_CLASS
+#define TEST_CLASS class
+#endif
+
+// Override this to test protected
+#ifndef TEST_PRIVATE
+#define TEST_PRIVATE private
+#endif
+
+TEST_CLASS
+__attribute__((__swift_attr__("private_fileid:main/blessed.swift"))) MyClass {
```

Comment:
Would be nice to have an annotated template to make sure this is propagated to the instantiations properly.

Reply:
Addressed in [f41af53](https://github.com/swiftlang/swift/pull/79093/commits/f41af53c18fdd0c263293b3ec943172817867793)